composer.json

```diff
@@ -36,9 +36,13 @@
         "ruflin/elastica": "~5.3",
         "symbiote/silverstripe-queuedjobs": "~4.0"
     },
+    "require-dev": {
```

Comment:
Should we add a .travis.yml while we're at it? ;-)

Reply:
Done, example build via our fork here: https://travis-ci.org/creative-commoners/silverstripe-elastica/builds/515513180